packages/ui-lib/src/Form/molecules/ButtonsBundle.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import ButtonWithIcon from '../atoms/ButtonWithIcon';
+import Button from '../atoms/Button';
+import { IButtonProps } from '..';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  button {
+    text-wrap: nowrap;
+  }
+`;
+
+export type IButtonType = 'default' | 'icon-button'
+
+export interface IBButton extends IButtonProps {
+  id: string
+  buttonType?: IButtonType
+  icon?: string
+  iconPosition?: 'left' | 'right'
+  text: string
+}
+
+export interface IButtonsBundle {
+  buttons : IBButton[]
+}
+
+const ButtonsBundle : React.FC<IButtonsBundle>= ({buttons}) => {
```

Comment:
Would you be opposed to renaming this to `ButtonStack`? I feel that would be a slightly clearer name descriptively.